src/main/java/uk/gov/hmcts/cmc/claimstore/services/OffersService.java

```diff
@@ -113,4 +114,15 @@ private void assertSettlementIsNotReached(Claim claim) {
     private String userAction(String userAction, String userType) {
         return userAction + "_" + userType;
     }
+
+    public Claim signSettlementAgreement(String externalId, Settlement settlement, String authorisation) {
+        final Claim claim = claimService.getClaimByExternalId(externalId, authorisation);
+        assertSettlementIsNotReached(claim);
```

Comment:
could you add some line breaks to make this more readable please?

i.e. here, 
after updateSettlement and after the app insights call

Reply:
done